dawn/renderer/components/Editor.js

```diff
@@ -29,10 +27,27 @@ import 'brace/theme/solarized_dark';
 import 'brace/theme/solarized_light';
 import 'brace/theme/terminal';
 
+import UpdateBox from './UpdateBox';
+import ConfigBox from './ConfigBox';
+import ConsoleOutput from './ConsoleOutput';
+import EditorButton from './EditorButton';
+
+const Client = require('ssh2').Client;
+
 const dialog = remote.dialog;
 const currentWindow = remote.getCurrentWindow();
 
 class Editor extends React.Component {
+  static pathToName(filepath) {
+    if (filepath !== null) {
+      if (process.platform === 'win32') {
+        return filepath.split('\\').pop();
+      }
+      return filepath.split('/').pop();
+    }
+    return '[ New File ]';
```

Comment:
pathToName is used in two contexts; one for file location and other for displaying file name above the Editor. Will change.